docs/adr/0016-updating-non-wmde-software.md

```diff
@@ -0,0 +1,39 @@
+- Software which is maintained by the Wikimedia Foundation (WMF), e.g. Wikidata Query Service (WDQS); in case of docker images also MediaWiki software.
+- Broadly-used software which is maintained by non-Wikimedia parties, e.g. Elastic Search, MySQL.
+- Wikibase/Wikidata-specific software which is maintained by Wikimedia community (volunteer) developers, e.g. Quick Statements.
+
+All these software components receive updates from their maintainers in a way which is not necessarily synchronized with the release cycle of Wikibase Suite.
+Users of Wikibase Suite might benefit from those changes, in particular from the updates which fix the incorrect functionality (bugs), and issues related to the software security.
+
+Offering updates to software components not maintained by the WMDE that are known to compatible with the rest of the Wikibase Suite would require applying updates to versions of the components included in the Wikibase Suite, possible changes to Wikibase Suite structure (e.g. configuration) when applicable, and necessary testing procedures to ensure continued functionality of the Wikibase Suite with the updated components.
+
+There might also be changes desired by the users of Wikibase Suite to be made to these software components (e.g. new functionality, bug fixes) that might be not be provided by the maintainers in subjectively-perceived timely manner.
+
+Making changes to the software not maintained by WMDE would mean additional effort for WMDE's software developer teams, and likely require WMDE development teams gather additional expertise to be able to make good contributions to those software components.
+
+
+## Decision
+
+To ensure the security and integrity of systems running Wikibase Suite, WMDE will be releasing updated versions of the Wikibase Suite that will include fixes to significant security issues discovered in the software components not maintained by WMDE, once those fixes have been published by respective maintainers.
+
+Occassionally, WMDE might also publish releases of Wikibase Suite that will include updates to the components not maintained by WMDE that bring important additional functionality.
+
+In order to maximize the impact WMDE software development teams can bring for the users of Wikibase Suite, WMDE will generally not be making changes to the software components of the Wikibase Suite that are not maintained by WMDE.
+
+## Consequences
+
+1. WMDE establishes a process of incorporating and releasing fixes to severe security issues in the components not maintained by WMDE. See more on this in [ADR 15](0015-security-fixes-non-wmde-software.md).
+2. WMDE will monitor changes to the components of Wikibase Suite which are not maintained by WMDE to understand non-security-related changes happening in those components. The list of channels to follow by WMDE staff will be published by the internal use.
```

Comment:
It's been some years since i'm back on IRC. In the current form I'm using it with Quassel it's sort of useless for monitoring anything as it requires me to be logged in constantly. As i remember this was solved back in the days by using a [bnc "bouncer"](https://en.wikipedia.org/wiki/BNC_(software)) to store the messages on. What is the WMF/WMDE/GDPR approved way of achieving the same functionality today?

Reply:
not exactly using a bouncer but WMDE has a solution some people were using. I would hope that we can have find more "established" channels where maintainers inform on new versions not just updates on IRC. But if that was the case, then yes, we'll be able to solve this

Reply:
s/by/for/

Many software packages have mailing list that can be followed for this. For most things using github, you can also watch a project for releases (only), but we would need to coordinate setting that up (for whom?).